cost/aws/superseded_instances/aws_superseded_instances.pt

```diff
@@ -1,13 +1,13 @@
 name "AWS Superseded EC2 Instances"
 rs_pt_ver 20180301
 type "policy"
-short_description "Checks for AWS EC2 instance types that have been superseded and, optionally, updates the instance type. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/aws/superseded_instance) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+short_description "Checks for AWS EC2 instance types that have been superseded and, optionally, updates the instance type. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/aws/superseded_instances) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
This requires an update to the meta policy too

See - https://github.com/flexera-public/policy_templates/blob/8f680069f39cac0b94888190065bfed193f5cb65/cost/aws/superseded_instances/aws_superseded_instances_meta_parent.pt#L4 (and also line 11)

You can actually run the following GitHub Actions workflow on this branch which will automatically generate the updates for the meta policy: https://github.com/flexera-public/policy_templates/actions/workflows/generate-meta-parent-policy-templates.yaml - 
![image](https://github.com/flexera-public/policy_templates/assets/92175447/dc20303a-1934-45fc-9624-e0a8a58d912d)

Reply:
Thank you, Nia.